services/order.py

```diff
@@ -0,0 +1,30 @@
+from typing import Optional
+
+from django.db import transaction
+from django.utils.timezone import now
+from db.models import Order, Ticket, User, MovieSession
+
+
+@transaction.atomic
+def create_order(tickets: list[dict],
+                 username: str,
+                 date: Optional[str] = None) -> Order:
+    order_date = date if date else now()
+    user_instance = User.objects.get(username=username)
+    order = Order.objects.create(user=user_instance, created_at=order_date)
+    tickets_to_create = [
+        Ticket(
+            row=ticket["row"],
+            seat=ticket["seat"],
+            movie_session=MovieSession.objects.get(id=ticket["movie_session"]),
```

Comment:
Consider adding error handling for the `MovieSession.objects.get(id=ticket['movie_session'])` call to manage cases where a movie session with the given ID does not exist. This can prevent the function from raising a `DoesNotExist` exception.